cobalt/media/base/cval_stats_test.cc

```diff
@@ -71,7 +57,8 @@ TEST(MediaCValStatsTest, NothingRecorded) {
   CValStats cval_stats_;
 
   cval_stats_.StartTimer(MediaTiming::SbPlayerCreate, kPipelineIdentifier);
-  sleep_ms(kSleepTimeMs);
+  SbThreadSleep(kSleepTimeMs);
```

Comment:
SbThreadSleep expects SbTime. I think the constant should reflect that.

Kinda funny that the tests still passed with the sleep reduced by 99.9%

Reply:
updated with SbTime.
Tests just confirm that if time has passed, a change is recorded, so thats understandable about them still passing with a different time.